lib/cocoapods/validator.rb

```diff
@@ -39,6 +39,7 @@ class Validator
     #         the platforms to lint.
     #
     def initialize(spec_or_path, source_urls, platforms = [])
+      @results = []
```

Comment:
where is this used?

Reply:
I was going to use this for stubbing in tests (tests would break since this isn't initialized until `validate` is called) but I ended up not using it. I can remove it